source/administration/object-management/object-versioning.rst

```diff
@@ -266,6 +265,129 @@ enable versioning on only a prefix or subset of objects in a bucket.
 Objects created prior to enabling versioning have a 
 ``null`` :ref:`version ID <minio-bucket-versioning-id>`.
 
+
+Exclude a Prefix From Versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can exclude certain :ref:`prefixes <minio-admin-concepts-organize-objects>` from versioning using the :ref:`MinIO Client <minio-client>`.
+This is useful for Spark/Hadoop workloads or others that initially create objects with temporary prefixes. 
+
+.. admonition:: Object locking
+   :class: note
+
+   Buckets with :ref:`object locking enabled <minio-object-locking>` require versioning and do not support excluding prefixes.
+
+- Use :mc-cmd:`mc version enable` with the :mc-cmd:`~mc version --excluded-prefixes` option:
+
+  .. code-block:: shell
+     :class: copyable
+
+     mc version enable --excluded-prefixes ALIAS/BUCKET "prefix1, prefix2"
+
+  - Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+  - Replace ``BUCKET`` with the :mc-cmd:`target bucket <mc version ALIAS>` you want to exclude :ref:`prefixes <minio-admin-concepts-organize-objects>` for.
```

Comment:
I tried to both follow the model of the existing example above, and link to something useful about prefixes/folders. Ideas? This seems a bit awkward.

Reply:
Couple of options here.

## Link to the S3 Docs
`Bucket` is a standard s3 concept. We could link to their docs explanation.
If you look in conf.py under extlinks, we have a role for that:
```
:s3-docs:`UsingBucket.html`
```

Basically, s3-docs adds the base URL up through `/latest/userguide/` then attaches whatever we have for the rest of the URL.

## Link to the list command

We could link to the mc list command:
```
:mc:`mc ls`
```

(That would list all buckets and objects at the root of the alias.)

## Link to the admin/concept.rst page

There's a section on that page about organizing objects within folders in buckets. Not ideal, but closest we have there. You'd need to add a directive to that page."

## My preference
I think I prefer linking to the S3 docs, personally.